Src/VsVimShared/Implementation/Misc/FallbackKeyProcessor.cs

```diff
@@ -158,7 +177,7 @@ private bool TryConvertLetterToKeyInput(KeyEventArgs keyEventArgs, out KeyInput
             }
 
             var c = (char)('a' + (key - Key.A));
-            keyInput = KeyInputUtil.ApplyKeyModifiersToChar(c, keyModifiers);
+            keyInput = KeyInputUtil.ChangeKeyModifiersDangerous(KeyInputUtil.CharToKeyInput(c), keyModifiers);

```

Comment:
Why did you need to make this change?  In general `ChangeKeyModifiersDangerous` should never be called because it produces key combinations that don't map to non-English languages.